doc/chapters/service-manifests/index.rst

```diff
@@ -0,0 +1,65 @@
+
+.. _service-manifests:
+
+Service manifests
+*****************
+
+Service manifests are JSON files mapping capabilities to gateway
+configurations. In other words, service manifests describe what
+exact gateway configurations are needed to configure the relevant
+gateways for a specific service in order to realize a capability.
+The service manifests contain the definitions of all the capabilities
+relevant for a service.
+
+For example, a platform might have a capability `com.acme.SomeResource`
+defined. This capability requires access to service `A`. Service `A`
+will then need to deploy a service manifest which contains all gateway
+configurations needed for the `com.acme.SomeResource` capability.
+
+A service manifest might also contain gateway configurations for more
+capabilities.
+
+More services than one might be involved to realize a capability and
+then each relevant service would need to be integrated in this way.
+
+For information about how to work with the manifest files, e.g. where to
+install them etc., see :ref:`Integration <integration>`.
+
+Manifest format
+===============
+
+A service manifest is a text file containing JSON. It is required to define
+a version set to "1", and an array "capabilities" which contains a
+"name" key with the capability name as value, and a "gateways" key which
+contains an array of objects each defining a gateway configuration identified
+with a gateway ID.
+
+Examples
```

Comment:
I will try to put a more clear explanation in the beginning what the reader can expect to find in this text. I'd like to have integration guidelines separate.